tensorflow/core/kernels/data/experimental/data_service_dataset_op.h

```diff
@@ -46,7 +46,9 @@ class IterationCounter : public ResourceBase {
 // Creates a dataset for reading from the tf.data service.
 class DataServiceDatasetOp : public DatasetOpKernel {
  public:
-  static constexpr const char* const kDatasetType = "DataService";
+//Tom: Avoid declare static constant variables in class, which will result in import error (Symbol not found in flat namespace) in macOS. 
```

Comment:
This looks like a good change if the constants are only used in the .cc file. No need to keep the old code as a comment.

Reply:
I'll remove the comments.